pkg/infra/iac2/find_imports.scm

```diff
@@ -0,0 +1,3 @@
+(program
```

Comment:
Technically these as TS files and not JS so it likely doesn't apply. Also, since these only check our code, we can be more rigid about the types of import statements we use.